### PR TITLE
Project Updates for RT-STPS 7.0 Patch 1 and IPOPP 4.1

### PR DESCRIPTION
**Pull request by alexnelsone**
*Issue #, if available:*

*Description of changes:*
Updated project for use of RT-STPS 7 Patch1.
Updated project for use of IPOPP 4.1
Updated README 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -269,17 +270,17 @@ Execute these commands on your local machine command line.
...
 ```
 
 ### Windows
 
 ```bash
-aws s3 cp %WORKING_DIR%\aws-groundstation-eos-pipeline\bash\ipopp-ingest.sh s3://%S3_BUCKET%/software/IPOPP/ipopp-ingest.sh --region %REGION% 
-aws s3 cp %WORKING_DIR%\aws-groundstation-eos-pipeline\bash\install-ipopp.sh s3://%S3_BUCKET%/software/IPOPP/install-ipopp.sh --region %REGION% 
-aws s3 cp %WORKING_DIR%\IMAPP_3.1.1_SPA_1.4_PATCH_2.tar.gz s3://%S3_BUCKET%/software/IMAPP/IMAPP_3.1.1_SPA_1.4_PATCH_2.tar.gz --region %REGION% 
-aws s3 cp %WORKING_DIR%\DRL-IPOPP_4.0_PATCH_1.tar.gz s3://%S3_BUCKET%/software/IPOPP/DRL-IPOPP_4.0_PATCH_1.tar.gz --region %REGION%
+aws s3 cp $WORKING_DIR/aws-groundstation-eos-pipeline/bash/ipopp-ingest.sh s3://${S3_BUCKET}/software/IPOPP/ipopp-ingest.sh --region $REGION
```
Comment (Viktor-Pankov) [inline README.md:280]: Why did you remove the \ from the Windows file paths?

**Review by Viktor-Pankov — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (alexnelsone) [reply README.md:280]: This was a mistake.  I will correct it.
